### PR TITLE
Update totalconnect to remove old icon

### PR DESCRIPTION
**Pull request by austinmroczek**
Update totalconnect to remove the old icon.  Also removed a reference to Honeywell since the Total Connect service appears to have been sold or made into a subsidiary with a different name.

## Proposed change

Update totalconnect to remove the old icon.  Also removed a reference to Honeywell since the Total Connect service appears to have been sold or made into a subsidiary with a different name.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/841
- This PR fixes or closes issue: https://github.com/home-assistant/brands/issues/713

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/totalconnect.markdown [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ---
-title: Honeywell Total Connect Alarm
-description: Instructions on how to integrate TotalConnect alarms into Home Assistant.
-logo: honeywell-tc.png
+title: Total Connect
+description: Instructions on how to integrate Total Connect into Home Assistant.
```
Comment (frenck) [inline source/_integrations/totalconnect.markdown]: These changes are not related to this PR... However, that is not a really big problem.
The problem is that changing this title has no effect. It will be overwritten and automatically synced from the integration manifest:

<https://github.com/home-assistant/core/blob/dev/homeassistant/components/totalconnect/manifest.json#L3>

Please revert those 2 lines here.

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by austinmroczek — COMMENTED** [review]
Unnecessary changes were reverted. Thanks for quick review.

**Review by frenck — APPROVED** [review]
Thanks for the quick resolution as well! 👍
